src/Kernel.php

```diff
@@ -139,4 +151,9 @@ public function loadRoutes(LoaderInterface $loader): RouteCollection
 
         return $collection;
     }
+
+    private function getAdditionalBundlesPath(): string
+    {
+        return \dirname($this->getProjectDir(), 3) . '/tests/TestApplication/config/bundles.php';
```

Comment:
Would be nice to pass the bundles file dir through envs

Reply:
I would be maybe for passing path to `tests/TestApplication` through env, or using by default this one 🤔